src/modality-specific-files/magnetic-resonance-spectroscopy.md

```diff
@@ -0,0 +1,260 @@
+| *J*-resolved spectroscopy                   | j           | Another 2D technique, where in a *J*-resolved acquisition, a series of transients are collected at different TEs. A 2D Fourier transform is applied to generate a 2D spectrum where one dimension characterizes both chemical shift and *J*-coupling and the other only *J*-coupling.                                                                                                                                                                                                                             |
+| Diffusion-weighted (DW) spectroscopy        | dw          | The diffusion of intracellular metabolites can be characterized using DW spectroscopy. In such acquisitions, the strength of gradients in a conventional MRS sequence is modulated to sensitize the metabolite signals to diffusion.                                                                                                                                                                                                                                                                              |
+| FID spectroscopy                            | fid         | FID spectroscopy is a pulse-acquire acquisition where an excitation pulse is followed by direct acquisition of the FID. This approach is most often used in MRSI (that is, FID-MRSI) when combined with slice- or slab-selection.                                                                                                                                                                                                                                                                                 |
+| Metabolite-cycled (MC) spectroscopy         | mc          | MC spectroscopy involves the use of asymmetric adiabatic inversion of the upfield and downfield parts of the MR spectrum, allowing for simultaneous acquisition of water and metabolite spectra.                                                                                                                                                                                                                                                                                                                  |
+| Spin-echo spectroscopy                      | spinecho    | An MRS experiment whereby the MR signal is detected using a spin-echo acquisition: 90°–180°–acq.                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+Each `<label>` in the table above MAY be combined with another to better describe the acquisition used.
+For example, `megaspecial`, `jpress`, `dwslaser`, `mcdwsteam`, and so on.
+
```

Comment:
In [examples](https://github.com/bids-standard/bids-examples/pull/425) a number is sometimes suffixed to the label (eg. megapress80/64). If that is part of the spec, I think it should be explained here.

Reply:
Hi @bpinsard. The `acq` label can take any form that a user wishes (see: https://bids-specification.readthedocs.io/en/bep022/appendices/entities.html#acq). The examples are just that: examples. Affixing a number is possible if it distinguishes acquisitions appropriately.

Reply:
Oh right! I thought it had a meaning like number of spectrum acquired or else.
I am not really versed in MRS and was mainly trying to understand how to BIDS-format some CMRR-megapress data. 
Just found in the README of that dataset:
> - TE/TR = 68 (for standard GABA editing) or 80 (for symmetric macromolecule suppression)/2000 ms

Using `acq-`  to store MR parameters seems a bit of a BIDS anti-pattern, but I'm not an MRS expert.
Anyway, thanks for the explanation!